src/servico/ProfessorService.java

```diff
@@ -67,34 +68,25 @@ public ProfessorService(){
         
         for(Professor professor : todosProfessores){
             
-            if(!professor.equals(professorAusente)){
-                //profsMenosAusente.add(professor);
+            if(!professor.equals(professorAusente)&&professor.EhCompativelCom(aulasPerdidas)){
                 
-                if(professor.EhCompativelCom(aulasPerdidas)){
-                    
-                    Boolean compatível = true;
+                Boolean compatível = true;
                     
-                    for(Ausencia ausenciaPreExistente : todasAsAusencias)
-                    {
-                        if(ausenciaPreExistente.getProfessorSubstituto() != null)
-                        {
-                            if(ausenciaPreExistente.getProfessorSubstituto().getNome().equals(professor.getNome()))
-                            {
-                                if(ausenciaPreExistente.getPeriodo().overlaps(ausência.getPeriodo()))
-                                    compatível = false;
+                for(Ausencia ausenciaPreExistente : todasAsAusencias){
+                    if(ausenciaPreExistente.getProfessorSubstituto() != null){
+                        if(ausenciaPreExistente.getProfessorSubstituto().getNome().equals(professor.getNome())){
+                            if(ausenciaPreExistente.getPeriodo().overlaps(ausência.getPeriodo())){

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 98).](https://www.codacy.com/app/calebepb/substituicao/file/2728288362/issues/source?bid=3308597&fileBranchId=3349138#l78)